src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -41,11 +41,12 @@
     <DocumentationFile>
     </DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)'!='Unix'">
+    <EnableNode>True</EnableNode>
+    <EnablePowerShell>True</EnablePowerShell>
+    <DefineConstants>$(DefineConstants);WINDOWS;FEATURE_NODE;FEATURE_POWERSHELL</DefineConstants>
```

Comment:
Let's simplify all this to just a single MONO flag. If we need additional complexity in the future for some reason we can add it later. That satisfies the requirements of this very temporary situation where we need to do compilation for Mac/Mono. When we move to .NET core this all goes away.